src/Microsoft.Health.Fhir.Liquid.Converter/Utilities/Hl7v2DataUtility.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Health.Fhir.Liquid.Converter.Utilities
+{
+    public class Hl7v2DataUtility
```

Comment:
Can we make this a static class? If so we don't have to new it before using its functions.